src/app/activity/bounty/List.ts

```diff
@@ -28,145 +27,131 @@ export const listBounty = async (request: ListRequest): Promise<any> => {
         customerId: request.guildId,
     });
 
-    const channel = await listUser.guild.channels.fetch(dbCustomerResult.bountyChannel);
-    const channelName = channel.name;
-
-
     Log.debug('Connected to database successfully.');
     Log.info('Bounty list type: ' + listType);
 
-	let IOUList: boolean = false;
+	let listTitle: string;
+	let openTitle = "Open";
 
 	switch (listType) { 
 	case 'CREATED_BY_ME':
-		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, isIOU: { $ne: true }, status: { $ne: 'Deleted' }, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
+		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, status: { $ne: 'Deleted' }, 'customerId': request.guildId }).sort({ status: -1, createdAt: -1 });
+		listTitle = "📝 Bounties Created by Me";
 		break;
 	case 'CLAIMED_BY_ME':
-		dbRecords = bountyCollection.find({ 'claimedBy.discordId': listUser.user.id, status: { $ne: 'Deleted' }, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		break;
-    case 'CLAIMED_BY_ME_AND_COMPLETE':
-        dbRecords = bountyCollection.find({ 'claimedBy.discordId': listUser.user.id, status: 'Completed', 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-        break;
-	case 'DRAFTED_BY_ME':
-		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, status: 'Draft', 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		break;
-	case 'OPEN':
-		dbRecords = bountyCollection.find({ status: BountyStatus.open, isIOU: { $ne: true }, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
+		dbRecords = bountyCollection.find({ $or: [ { 'claimedBy.discordId': listUser.user.id }, { applicants: { $elemMatch: { discordId: listUser.user.id }}} ] , status: { $ne: 'Deleted' }, 'customerId': request.guildId }).sort({ status: -1, createdAt: -1 });
+		listTitle = "👷 Bounties Claimed or Applied For by Me";
+		openTitle = "Applied For"
 		break;
-	case 'IN_PROGRESS':
-		dbRecords = bountyCollection.find({ status: BountyStatus.in_progress, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		break;
-	case 'PAID_BY_ME':
-		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, status: BountyStatus.complete, isIOU: true, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		IOUList = true;
-		break;
-	case 'UNPAID_BY_ME':
-		dbRecords = bountyCollection.find({ 'createdBy.discordId': listUser.user.id, status: BountyStatus.open, isIOU: true, 'customerId': request.guildId }).limit(DB_RECORD_LIMIT);
-		IOUList = true;
-		break;
-	}
-	if (!(await dbRecords.hasNext())) {
-		return await listUser.send({ content: 'We couldn\'t find any bounties!' });
+	default: 
+		dbRecords = bountyCollection.find({ $or: [ { status: BountyStatus.open } , { status: BountyStatus.in_progress }, { status: BountyStatus.in_review } ], isIOU: { $ne: true }, 'customerId': request.guildId }).sort({ status: -1, createdAt: -1 });
```

Comment:
I'd double check to see if we have any IOUs hanging around that haven't been completed. I don't think we do. If we don't, I think we can remove `isIOU: { $ne: true }`